integration-libs/opf/base/occ/adapters/occ-opf.adapter.ts

```diff
@@ -43,17 +45,25 @@ export class OccOpfPaymentAdapter implements OpfPaymentAdapter {
     protected config: OpfConfig
   ) {}
 
-  header: { [name: string]: string } = {
```

Comment:
The header situation is part of CXSPA-4688 that will be merged soon. You can ignore. You won't be able to test it without it.